playbooks/iscsi_monitoring_setup.yaml

```diff
@@ -166,19 +152,15 @@
 - name: Prepare test
   hosts: iscsi_initiator
   become: yes
+  vars:
+    script_name: iscsi_connection_status.sh
   tasks:
     - name: copy test script on initiator
       copy:
-        src: iscsi_connection_status.sh
+        src: "iscsi/{{ script_path }}"
```

Comment:
Where `script_path` variable is defined?

Reply:
fixed